packages/drink-buddy-ui/prisma/schema.prisma

```diff
@@ -10,7 +10,57 @@ datasource db {
   url      = "postgresql://postgres:Admin0@localhost:5432/postgers" //TODO: env("DATABASE_URL")
 }
 
+model User {
+  id                           Int      @id @default(autoincrement())
+  createdAt                    DateTime @default(now())
+  updatedAt                    DateTime @updatedAt
+  username                     String   @unique
+  email                        String
+  drinks                       Drink[]
+  someFieldUsedForExternalAuth String //will be handeled later
+}
+
 model Drink {
-  id  String @id @default(uuid())
-  name String
-}
\ No newline at end of file
+  id                Int             @id @default(autoincrement())
```

Comment:
Same as above - i also think that we should have different ID length for USERS / PRODUCTS / INGRIDIENTS to easily distinguish one from another - where ingridient id should be the shortest - keep it alphnumeric i think 4 chars ID is more than enough for it

Reply:
individual products and users should have the longest ones
i thing in 8-12 length category

Reply:
if you count alphanumeric case sensitive input you have $62^{n}$ unique ones which is a big difference if you compare it to usinjg numbers only